src/azure-cli/azure/cli/command_modules/databoxedge/generated/_help.py

```diff
@@ -0,0 +1,332 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+
+from knack.help_files import helps
+
+
+helps['databoxedge device'] = """
+    type: group
+    short-summary: Manage device with databoxedge
+"""
+
+helps['databoxedge device list'] = """
+    type: command
+    short-summary: "Gets all the Data Box Edge/Data Box Gateway devices in a resource group. And Gets all the Data Box \
+Edge/Data Box Gateway devices in a subscription."
+    examples:
+      - name: DataBoxEdgeDeviceGetByResourceGroup
+        text: |-
+               az databoxedge device list --resource-group "GroupForEdgeAutomation"
+      - name: DataBoxEdgeDeviceGetBySubscription
+        text: |-
+               az databoxedge device list
+"""
+
+helps['databoxedge device show'] = """
+    type: command
+    short-summary: "Gets the properties of the Data Box Edge/Data Box Gateway device."
+    examples:
+      - name: DataBoxEdgeDeviceGetByName
+        text: |-
+               az databoxedge device show --name "testedgedevice" --resource-group "GroupForEdgeAutomation"
+"""
+
+helps['databoxedge device create'] = """
+    type: command
+    short-summary: "Create a Data Box Edge/Data Box Gateway resource."
+    parameters:
+      - name: --sku
+        short-summary: "The SKU type."
+        long-summary: |
+            Usage: --sku name=XX tier=XX
```

Comment:
consider if we could specify sku with only one property.

Reply:
The sku name can be 'Gateway' or 'Edge', the tier can be 'Standard' or something else. How can we specify with only one property?